packages/api-specification/src/hiddenWindow.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
```

Comment:
I think it's okay for now - at some point I expect we'll need to split up the demo/code from the specification to make it a bit easier to maintain.

> I think hiddenMainWindow is probably my preference from those.

:+1: